Cargo.toml

```diff
@@ -248,6 +248,10 @@ move-vm-ext = { path = "types/move-vm-ext" }
 num-derive = "0.4.2"
 num-traits = "0.2.14"
 once_cell = "1.8.0"
+opentelemetry = { version = "0.26" }
```

Comment:
@mzabaluev can this PR be merged soon?

I'm adding OpenTelemetry to the bridge so I'll need the same deps.

Reply:
Let me resolve the conflicts with latest main and we can bump it up for review.